1.9/administration/debugging/quickstart.md

```diff
@@ -0,0 +1,241 @@
+
+    1.  Create the following app definition and save as `⁠⁠⁠⁠test-job.json`. This specifies a sleep job that runs for `10000000` seconds.
+    
+        ```bash
+        {
+          "id": "task-exec-test",
+          "labels": {},
+          "run": {
+            "artifacts": [],
+            "cmd": "sleep 100000000",
+            "cpus": 0.01,
+            "disk": 0,
+            "env": {},
+            "maxLaunchDelay": 3600,
+            "mem": 32,
+            "placement": {
+              "constraints": []
+            },
+            "restart": {
+              "policy": "NEVER"
+            },
+            "volumes": []
+          }
+        }
+        ```
+    
+    1.  Deploy the job app with this CLI command:
+    
+        ```bash
+        $ dcos job add test-job.json
+        ```
+        
+    1.  Verify that the app has been successfully deployed:
+    
+        ```bash
+        $ dcos job list
+        ID              DESCRIPTION  STATUS   LAST SUCCESFUL RUN        
+        task-exec-test               Unscheduled         None
+        ```
+
+    1.  Run the job:
+    
+        ```bash
+        $ dcos job run task-exec-test
+        ```
+
+1.  Get the task ID of the job with this CLI command:
+
+    ```bash
+    $ dcos task
+    ```
+    
+    The output should look similar to this: 
+    
+    ```bash
+    NAME                                HOST       USER  STATE  ID                                                                       
+    20161209183121nz2F5.klueska-test    10.0.2.53  root    R    <task_id>
+    ```
+
+1.  Launch a TTY process inside of the container with the task ID (`<task_id>`) specified. This will launch an interactive Bash session.
+
+    ```bash
+    $ dcos task exec --interactive --tty <task_id> bash
+    ```
+    
+    You should now be inside the container running an interactive Bash session.
+    
+    ```bash
+    root@ip-10-0-2-53 / #
+    ```
+    
+    **Tip:** You can use shorthand abbreviations `-i` for `--interactive` or `-t` for `--tty`. Also, only the beginning unique characters of the `<task_id>` are required. For example, if your task is named `exec-test_20161214195` and there are no other task IDs that begin with the letter `e`, this is valid command syntax: `dcos task exec -i -t e bash`. For more information, see the CLI command [reference](/docs/1.9/usage/cli/command-reference/).
+    
+1.  Run a command from the interactive Bash session from inside the cluster. For example, the `ls` command:
+
+    ```bash
+    root@ip-10-0-1-104 / # ls
+    bin   dev  home  lib64	     media  opt   root	sbin  sys  usr
+    boot  etc  lib	 lost+found  mnt    proc  run	srv   tmp  var
+    ```
+
+# Pipe output from a command running inside a container 
+
+You can run commands inside a container by using the `dcos exec` command. In this example, a long running Marathon app is launched and then the `dcos task exec` command is used to get the hostname of the node running your app. 
```

Comment:
It's `dcos task exec` not `dcos exec`

Also there is a mismatch in voice here. You say "is launched" followed by "your app". I would probably change the latter to simply "the app".